projects/jdwp/codegen/new_type_generator.py

```diff
@@ -0,0 +1,21 @@
+"""New type Generator."""
+from projects.jdwp.defs.schema import PrimitiveType
+
+
+def get_python_type(jdwp_type: str) -> str:
```

Comment:
Perhaps you could take argument of type `PrimitiveType` instead ? It's more descriptive than `str` and admits less incorrect values.